crawler/src/main/java/com/brandwatch/minibcr/crawler/crawlerjob/Crawler.java

```diff
@@ -0,0 +1,21 @@
+package com.brandwatch.minibcr.crawler.crawlerjob;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import com.brandwatch.minibcr.common.domain.Resource;
+import com.brandwatch.minibcr.crawler.services.CrawlerService;
+
+@Component
+public class Crawler {
+
+    @Autowired
+    CrawlerService crawlerService;
+
+    @Scheduled(fixedDelay = 5000)
```

Comment:
Any reason why you chose 5 seconds?

Reply:
Just to see that it works, instead of waiting a couple of mins 😬.